src/temperature/models.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer, DateTime, Float
+
+from src.database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    city_id = Column(Integer)
```

Comment:
Consider adding a foreign key constraint to the `city_id` column to ensure referential integrity with the `City` table. This will help maintain data consistency and enforce relationships between tables.